contrib/prometheus-watcher/examples/prometheus-watcher.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-watcher-example
+data:
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus-watcher
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: prometheus-watcher
+    spec:
+      volumes:
+      - name: rules
+        emptyDir: {}
+      - name: config
+        configMap:
+          name: prometheus-watcher-example
+      containers:
+      - name: ubuntu
+        image: ubuntu
+        command: ["sleep", "3600"]
+        volumeMounts:
+        - name: rules
+          mountPath: /etc/prometheus/rules
+          readOnly: true
+      - name: prometheus-watcher
+        image: quay.io/coreos/prometheus-watcher:v0.0.1
+        args:
+        - '-config-volume-dir=/etc/prometheus/config'
+        - '-rule-volume-dir=/etc/prometheus/rules'
+        - '-reload-url=http://localhost:9090/-/reload'
+        resources:
+          requests:
+            memory: "16Mi"
+            cpu: "50m"
+          limits:
+            memory: "32Mi"
+            cpu: "100m"
```

Comment:
True, I'll run it in the kube-prometheus environment and replace with something that I'll see from metrics.